### ISSUE TITLE
Document migration from pkg_web

### ISSUE THREAD

**Issue by alexeagle** [aspect-build/rules_js#271]
I think it's some combo of "use some other packaging rule from rules_pkg" and #254. Maybe we can expose a macro that is a close approximation.

**Comment by ss185994** [comment]
is there an appropriate way to migrate from `pkg_web` in `rules_nodejs` to `rules_js`?

**Comment by alexeagle** [comment]
In migrations I've been doing, I simply removed `pkg_web` since the bundler produced a directory that could be directly deployed.

Can you give an example of how you're using it - what inputs you give it and what output you expect? I think rules_pkg should cover most use cases.

**Comment by ss185994** [comment]
this Is how it was setup under `rules_nodejs`

cmd/web/provider
```
container_image(
    name = "some-image",
    base = ":some-container",
    files = ["//path/to/web/iam"],
    ports = ["8080"],
    tags = ["manual"],
)
```

cmd/web/iam
```
pkg_web(
     name = "iam",
     srcs = [":build"], // result of react-scripts build
     substitutions = {"placeholder": "some_semver"},
     visibility = ["//visibility:public"],
 )
```

ive gotten the react-cra example more or less working would it be as easy as tossing the result of [build](https://github.com/aspect-build/bazel-examples/blob/1b8be767c587bc1187efa283af515f4c78e78b86/react-cra/BUILD.bazel#L23) to the `container_image`?

**Comment by alexeagle** [comment]
Yes exactly your pkg_web might not be doing anything extra there. Were you relying on it to stamp something? You can just compare the container layers before and after to convince yourself it produces the same result (or identify what needs to be changed to get it to match)
